src/main/java/com/faforever/client/mod/ModVaultController.java

```diff
@@ -352,6 +358,18 @@ public void onLoadMoreButtonClicked() {
         });
   }
 
+  public void manageMods() {
+    ModManagerController modManagerController = uiService.loadFxml("theme/mod_manager.fxml");
+    List<Action> actions = Arrays.asList(new Action(i18n.get("modVault.modManager.cancel"), Type.OK_DONE, Event::consume),
+        new Action(i18n.get("modVault.modManager.save"), event -> modManagerController.apply()));
+    notificationService.addNotification(new ImmediateNotification(i18n.get("modVault.modManager"),
+        i18n.get("modVault.modManager.description"),
+        Severity.INFO,
+        actions,
+        modManagerController.getRoot()
+    ));
```

Comment:
Is this to avoid the JFXDialog in UIService.showInDialog?
Otherwise I am not sure why you use this over the UIService.

Reply:
No idea either. Maybe should do that.

Reply:
Ok well I know now. Because then i need yet another Controller I guess cause I need a title and a cancle and save button. Not sure if uiService offers that

Reply:
Hmm yeah I see what you mean since the UIService doesn't auto add the buttons. So it makes sense. Maybe a little off because it is not really a notification but not a big deal.

Reply:
Well used uiservice now